docs/tutorials/dronecan_firmware_upgrade.rst

```diff
@@ -0,0 +1,134 @@
+will be used in each of the following examples. See the DroneCAN GUI tool's documentation to find more information on supported devices.
+
+DroneCAN GUI Tool Basics
+============================
+After opening the DroneCAN GUI tool, you are met with the following screen: 
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/dronecan_gui_start_page.png
+
+Under *Select CAN Interface*, select the name of your connected USB-to-CAN device. In this case, the device is named USB Serial Device. 
+By default, all DroneCAN enabled Vertiq devices have a bitrate of 500000 bits per second. If you have not changed your module's bitrate, you must configure 
+*CAN bus bit rate* to 500000. Otherwise, you must set *CAN bus bit rate* to match the value set on your module. Select OK, and you will see the following window:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/dronecan_gui_main_page.png
+    :scale: 40%
+
+Here, you can view other devices detected on the bus, configure devices, and importantly here, trigger firmware updates. In order to interact with other DroneCAN nodes on the bus, 
+the USB-to-CAN node needs its own unique node ID. To set your device's node ID, find *Set local node ID* in the top left, pick a node ID unique from all other devices on the 
+bus (the default 127 is unique from the default Vertiq node ID of 99), and click the check mark.
+
+Now, connect your module to your USB-to-CAN device, and power it on. You should see it appear in the list of *Online nodes*:
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/module_seen.png
+
+Double click the module's node to find its DroneCAN node properties
+
+.. image:: ../_static/tutorial_images/dronecan_firmware_upgrade/module_properties.png
+    :scale: 80%
+
+From here, you can view the module's parameters by selecting *Fetch All*, you can reboot your module with *Restart*, or you can start a firmware update with *Update Firmware*. 
+In order to change a parameter's value, simply double click it, change the value, and click *Send*.
+
+.. _dronecan_upgrade_instructions:
+
+Updating Firmware with the DroneCAN GUI
+============================================
+Now that you have a basic understanding of the DroneCAN GUI's layout and interactions, we can move on to updating firmware through the GUI tool. First, you must download the correct firmware for your 
+module from `our website <https://www.vertiq.co/>`_. Take care to ensure that the firmware you download is meant for your module (size, Kv, firmware style, etc.). 
+Failure to do so can result in severe damage to your module. Inside of the downloaded .zip archive, you should find a file called combined.bin as well as a file called app.bin. 
+
+.. note::
+    If your zip archive does not include a file named app.bin, you cannot update your module's firmware via DroneCAN. An upcoming firmware release for your module 
+    will make the app.bin file available.
```

Comment:
Might just alter that second sentence a bit: "Future firmware releases will include app.bin files for all DroneCAN capable modules"